folly/Range.h

```diff
@@ -474,7 +474,7 @@ class Range {
   template <typename ValueType>
   struct StringViewType
       : std::conditional<
-            std::is_pod<std::remove_const_t<ValueType>>::value,
+            std::is_trivially_copyable<std::remove_const_t<ValueType>>::value,
```

Comment:
Need to use `folly::is_trivially_copyable` here.

Reply:
```suggestion
            folly::is_trivially_copyable<std::remove_const_t<ValueType>>::value,
```